### PR TITLE
Create Discussions

### PR DESCRIPTION
**Pull request by ashukla13**
Added a file that describes how SIG members can participate in Discussions

### CODE REVIEW

File: docs/Discussions [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## DevOps Mutualization SIG Discussions
+
+DevOps Mutualization SIG uses Github Discussions that are only visible to members who are part of the DevOps SIG [team](https://github.com/orgs/finos/teams/devops-mutualization-participants/) as agreed upon in the [meeting on Feb 18th 2020](https://github.com/finos/devops-mutualization/issues/18)
```
Comment (mcleo-d) [inline docs/Discussions:3]: `Github Discussions` **should be** `[Github Team Discussions](https://odp.finos.org/docs/project-collaboration#github-team-discussions)`

Comment (mcleo-d) [reply docs/Discussions:3]: `members who are part of the DevOps SIG [team](https://github.com/orgs/finos/teams/devops-mutualization-participants/)` **should be** `members who are part of the [devops-mutualization-participants team](https://github.com/orgs/finos/teams/devops-mutualization-participants/)`

File: docs/Discussions [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Discussions augments  Issues in this repository for public discussions. 
+
+## How to Participate in DevOps SIG Discussions
+- If you are not in the FINOS Github org, email help@finos.org and request that your Github account is added.
```
Comment (mcleo-d) [inline docs/Discussions:7]: `If you are not in the FINOS Github org` **should be** `If you're are not a member of the [FINOS Github Organisation](https://github.com/orgs/finos/people)`

Comment (mcleo-d) [reply docs/Discussions:7]: `request that your Github account is added` **should be** `request an invitation to join from the FINOS team.`

File: docs/Discussions [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- Email Amol Shukla or Anders Wallgren so they can add you to the Github team for the DevOps SIG.
```
Comment (mcleo-d) [inline docs/Discussions:8]: `Email Amol Shukla or Anders Wallgren so they can add you to the Github team for the DevOps SIG.` **should be** `If you are a member of the FINOS GitHub Organisation, please email Amol Shukla or Anders Wallgren so they can add you to the devops-mutualization-participants team for DevOps Mutualization SIG discussions.`

File: docs/Discussions [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+DevOps Mutualization SIG uses Github Discussions that are only visible to members who are part of the DevOps SIG [team](https://github.com/orgs/finos/teams/devops-mutualization-participants/) as agreed upon in the [meeting on Feb 18th 2020](https://github.com/finos/devops-mutualization/issues/18)
+Discussions augments  Issues in this repository for public discussions. 
+
+## How to Participate in DevOps SIG Discussions
+- If you are not in the FINOS Github org, email help@finos.org and request that your Github account is added.
+- Email Amol Shukla or Anders Wallgren so they can add you to the Github team for the DevOps SIG. 
+- You can now view or see Discussions
+
+
```
Comment (mcleo-d) [inline docs/Discussions:11]: **Please add** : `In order to collaborate in FINOS Team Discussions, you must have a [GitHub Account](https://github.com/) and have signed a [FINOS Contributor License Agreement (CLA)](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Contribution+Compliance+Requirements#ContributionComplianceRequirements-ContributorLicenseAgreement). Please email help@finos.org for help and more information.`

**Review by mcleo-d — COMMENTED** [review]
Hi @ashukla13 - I have done my review and added my updates inline 👍 

Let me know if you need help updating.

James.

**Comment by ashukla13** [comment]
thanks for your feedback, @mcleo-d; please re-review

**Comment by mcleo-d** [comment]
@ashukla13 - Thanks so much for the work done here. I have augmented text from the FINOS ODP project to help introduce the concept to GH Team Discussions. I also updated the workflow a little too.

**Pull request merged**